internal/integration/cli/apply-config.go

```diff
@@ -0,0 +1,53 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+//go:build integration_cli
+// +build integration_cli
+
+package cli
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/talos-systems/talos/internal/integration/base"
+)
+
+// ApplyConfigSuite verifies dmesg command.
+type ApplyConfigSuite struct {
+	base.CLISuite
+}
+
+// SuiteName ...
+func (suite *ApplyConfigSuite) SuiteName() string {
+	return "cli.ApplyConfigSuite"
+}
+
+// TestApplyWithPatch verifies that .
+func (suite *ApplyConfigSuite) TestApplyWithPatch() {
+	patch := `---
+cluster:
+  apiServer:
+    extraArgs:
+      logging-format: text`
+
+	node := suite.RandomDiscoveredNodeInternalIP()
```

Comment:
I think it should be machine.TypeControlPlane, otherwise the patch doesn't make much sense?

Reply:
yeah... makes sense. Fixed